salesforce-composite/2.4/modules/ROOT/pages/salesforce-composite-connector-reference.adoc

```diff
@@ -58,7 +58,7 @@ Release Notes: xref:release-notes::connector/salesforce-composite-connector-rele
 ** SECONDS
 ** MINUTES
 ** HOURS
-** DAYS |  A TimeUnit that qualifies the #connectionTimeout |  SECONDS | 10
+** DAYS |  A TimeUnit that qualifies the #connectionTimeout |  SECONDS | 
```

Comment:
A time unit that qualifies the **Connection Timeout** field.
or
A time unit that qualifies **Connection Timeout**.

Reply:
Why is there a number sign before connectionTimeout?